config/routes.rb

```diff
@@ -77,6 +77,7 @@
         put 'clone'
         put 'publish'
         get 'choose_questions', :path => 'choose-questions'
+        resources :qing_groups, :only => :new
```

Comment:
Can we add `:path => 'qing-groups'` for consistency here? Not that the user will ever see this URL in the addy bar...